README.md

```diff
@@ -11,50 +11,139 @@ It provides a robust, strongly-typed interface for efficient development, with s
+| ParentFilePrefix | string | The `parent_file_prefix` parameter is used to specify a virtual folder destination for the uploaded files, diverging from the default 'Inbox' folder. By setting this parameter, users can organize files into specific virtual directories, enhancing file management and accessibility within Alfred's system. |
+
+**Example:**
 
    ```csharp
    // Upload remote file
    var uploadResult = await alfred.File.UploadAsync(new FileUploadRequest
    {
-       Urls = new List<string> { "http://example.com/file.pdf" },
+       Urls = new List<string> { "https://pdfobject.com/pdf/sample.pdf" },
        Metadata = "{'DocumentType':'Invoice'}",       
    });   
    ```
 
+#### Upload File from Stream
+
+This method allows you to upload a file from a stream to Alfred and associate it with a session ID this method will not trigger the job processing this method is ideal for uploading files that are part of a larger job.
+
+   | Parameter | Type | Description |
+| --- | --- | --- |
+| FileStream | Stream | Stream of the file to upload. |
+| Filename | string | Unique name of the file within an object storage source. |
+| SessionId | Guid | Session ID to associate with the file. |
+| Metadata | string | JSON object or JSON array of objects containing metadata fields for a given remote file. |
+
+**Example:**
+
+   ```csharp
+   // Upload file from stream
+   Stream stream = new StreamReader("file_path\\sample.pdf").BaseStream;
+Guid sessionId= (await alfred.DeferredSession.CreateAsync()).SessionId;
+
+var response = await alfred.File.UploadFileAsync(new UploadFileRequest { FileStream = stream, FileName = "sample.pdf", SessionId = sessionId, Metadata = { } });
+Console.WriteLine(response);
+   ```
+
+## Alfred
+
+Alfred is a powerful document processing platform that enables you to extract, index, and search through large document collections with ease. It offers a wide range of features, including:
+
+- **Indexing**: Powerful indexing engine that can index and search through millions of documents in seconds.
+
+- **Extraction**: Can extract specific data from PDFs, images, and other documents with ease using its powerful extraction engine.
+
+- **Tagging**: Tag documents based on their content, making it easy to organize and search through large document collections.
+
+- **Job Management**: Provides a robust job management system that allows you to schedule and monitor document processing jobs.
+
+- **Integration**: Alfred can be easily integrated into your existing applications using its powerful API and SDKs.
+
+- **Scalability**: Alfred is designed to scale with your needs, whether you're processing thousands of documents a day or millions.
```

Comment:
I don't think this section should go at the end. I believe @rroa previous comments highlighted the features (bullet point) ordering (not the order in which this section appears within the whole README.md). The features list should effectively be ordered to reflect Alfred's file's journey in order to make it easier to catch.